frontend/src/app/libs/auth/services/auth-repository.service.spec.ts

```diff
@@ -63,4 +70,14 @@ describe('AuthService', () => {
 			httpRequest.flush(validResponse);
 		});
 	});
+
+	describe('LoadUser', () => {
+		it('should return an Observable<UserDto>', () => {
+			authRepository.loadUser().subscribe((userDtoResponse: UserDto) => {
+				expect(userDtoResponse).toEqual(userDto);
+			});
+			const httpRequest: TestRequest = httpMock.expectOne('/api/users/self');
```

Comment:
lets avoid http mock in unit testing, you can mock whole httpservice and stub it's `.get` method